src/components/Homepage/Leadspace/Leadspace.styles.js

```diff
@@ -1,6 +1,6 @@
 const styles = {
     root: {
-        height: 'calc(100vh - 4rem)',
+        height: 'calc(100vh - 3rem)',
```

Comment:
Note: This subtracts the nav height so the hero image covers 100% remaining height of viewport